hztb-rest-api/src/main/java/com/svs/hztb/api/sm/model/group/GroupOutput.java

```diff
@@ -8,28 +8,33 @@
 import com.svs.hztb.api.sm.model.opinion.Status;
 
 public class GroupOutput extends BasicOutput {
-	    private Status status;
-		private ErrorOutput errorOutput;
-		private List<GroupDetail> groupDetailList = new ArrayList<GroupDetail>();
-		public Status getStatus() {
-			int i=1;
-			
-			return status;
-		}
-		public void setStatus(Status status) {
-			this.status = status;
-		}
-		public ErrorOutput getErrorOutput() {
-			return errorOutput;
-		}
-		public void setErrorOutput(ErrorOutput errorOutput) {
-			this.errorOutput = errorOutput;
-		}
-		public List<GroupDetail> getGroupDetailList() {
-			return groupDetailList;
-		}
-		public void setGroupDetailList(List<GroupDetail> groupDetailList) {
-
-			this.groupDetailList = groupDetailList;
-		}
+	private Status status;
+	private ErrorOutput errorOutput;
+	private List<GroupDetail> groupDetailList = new ArrayList<GroupDetail>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS2293)